index.js

```diff
@@ -66,6 +67,55 @@ function safeJson(input) {
   }
 }
 
+/**
+ * @param {string} path
+ * @param {string} key
+ * @param {(string | boolean)} value
+ */
+function setPathOutput(path, key, value) {
+  if (path === '.') {
+    core.setOutput(key, value);
+  } else {
+    core.setOutput(`${path}--${key}`, value);
+  }
+}
+
+/**
+ * Add specific outputs for each release.
+ *
+ * Most notable is the `paths_released` output which
+ * makes it easy to build automations around monorepo setups.
+ */
+function outputReleases(releases) {
+  releases = releases.filter((release) => release !== undefined);
```

Comment:
Should we also filter `null`, and maybe other falsey values?

```suggestion
  releases = releases.filter(Boolean);
```

We can then remove the `if (!release) continue;`.